doc/architecture/README.md

```diff
@@ -29,4 +29,5 @@ To suggest an ADR, please make use of the [ADR template](./adr-template.md) prov
 | ------ | ----------- | ------ |
 | [001](./adr-001.md) | Add CosmWasm Module | Accepted |
 | [002](./adr-002.md) | Subscriptions in CosmWasm | Accepted |
-| [003](./adr-003.md) | Canis Major (1st Network Upgrade Scope of Breaking Changes) | Proposed |
\ No newline at end of file
+| [003](./adr-003.md) | Canis Major (1st Network Upgrade Scope of Breaking Changes) | Proposed |
```

Comment:
```suggestion
| [003](./adr-003.md) | Canis Major (1st Network Upgrade Scope of Breaking Changes) | Accepted |
```

Reply:
Do you want to change the status of ADR-003 as well?

Reply:
yes, it's already on the mainnet / released in v2